op-node/rollup/derive/channel_bank.go

```diff
@@ -78,10 +75,8 @@ func (ib *ChannelBank) prune() {
 // IngestData adds new L1 data to the channel bank.
 // Read() should be called repeatedly first, until everything has been read, before adding new data.\
 func (ib *ChannelBank) IngestData(data []byte) {
```

Comment:
The `ib` receiver name is legacy, if you're refactoring it anyway we can rename it to `cb` or something. Optional change

Reply:
done